tests/test_frequenz_dispatch.py

```diff
@@ -207,7 +198,7 @@ async def test_existing_dispatch_deleted(
         case Created(dispatch) | Updated(dispatch):
             assert False, "Expected a deleted event"
         case Deleted(dispatch):
-            sample.create_time = dispatch.create_time
```

Comment:
Create time is now set through line 191 `sample = ...` which returns the created dispatch.